XamarinCommunityToolkit/AttachedProperties/ProgressBarAttachedProperties.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.AttachedProperties
+{
+	public static class ProgressBarAttachedProperties
+	{
+		const string animatedProgress = "AnimatedProgres";
+
+		public static BindableProperty AnimatedProgressProperty = BindableProperty.CreateAttached(animatedProgress, typeof(double), typeof(ProgressBar), 0.0d, propertyChanged: ProgressBarProgressChanged);
+
+		static void ProgressBarProgressChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			var progressBar = bindable as ProgressBar;
+
+			if (progressBar == null)
+				return;
+
+			progressBar.ProgressTo((double)newValue, Convert.ToUInt32(Math.Max(0, 500)), Easing.Linear);
```

Comment:
What does this code do?)

```Convert.ToUInt32(Math.Max(0, 500))```
It compares 0 and 500 and selects max (always 500), then convert it to unsigned integer
Why not just write **500u** ?

Reply:
Damn... the rebase was one version to high in the commits. Because I already did changes for that last time. Let me double check it again.